server/src/main/java/org/apache/iotdb/db/engine/storagegroup/TsFileProcessor.java

```diff
@@ -317,7 +321,17 @@ void asyncClose() {
       logger.debug(FLUSH_QUERY_WRITE_LOCKED, storageGroupName, tsFileResource.getFile().getName());
     }
     try {
-      logger.info("Async close the file: {}", tsFileResource.getFile().getAbsolutePath());
+
+      if (workMemTable != null) {
+        logger.info(
+            "{}: flush a working memtable in async close tsfile {}, memtable size: {}, tsfile size: {}",
+            storageGroupName, tsFileResource.getFile().getAbsolutePath(), workMemTable.memSize(),
+            tsFileResource.getFileSize());
+      } else {
+        logger.info("{}: flush a NotifyFlushMemTable in async close tsfile {}, tsfile size: {}",
+            storageGroupName, tsFileResource.getFile().getAbsolutePath(), tsFileResource.getFileSize());
```

Comment:
I suppose `tsFileResource.getFileSize()` induces IO, so better to wrap with `if (logger.isInfoEnabled()){}`